bpfman.spec

```diff
@@ -4,18 +4,18 @@
 %global crate bpfman
 
 ## START DO NOT OVERRIDE: These are generated at runtime by .packit.sh
-%global commit 0
-%global shortcommit 0
-%global base_version 0
-%global prerelease 0
+%global commit 3b1aa1d0c42bd6f2dfef1d7cf27f4b7362307e49
```

Comment:
I'm not sure how this works, but should these values be filled in?

Reply:
Removed